apps/src/applab/DesignModeBox.jsx

```diff
@@ -59,6 +59,7 @@ module.exports = React.createClass({
           <div data-element-type="DROPDOWN" className="new-design-element">dropdown</div>
           <div data-element-type="IMAGE" className="new-design-element">image</div>
           <div data-element-type="CANVAS" className="new-design-element">canvas</div>
+          <div data-element-type="SCREEN" className="new-design-element">screen</div>
```

Comment:
Agreed. I also mentioned this to Sarah. I suspect we'll end up changing this in the future.